src/types/Submissions.d.ts

```diff
@@ -381,3 +381,5 @@ type SubmitButtonResult = {
   tooltip?: string;
   _identifier?: string;
 };
+
+type AccessType = "All" | "Controlled" | "Open";
```

Comment:
I think this is technically a Application.d.ts (Questionnaire) type? Or maybe just a ApprovedStudy.d.ts type